cxx-qt-gen/src/parser/qobject.rs

```diff
@@ -88,11 +88,11 @@ impl ParsedCxxQtData {
         Ok(match &item {
             Item::Enum(item_enum) => {
```

Comment:
Can you split up the code for the different match parts into separate functions, i.e.:
``` rust
pub fn parse_cxx_qt_item(&mut self, item: Item) -> Result<Option<Item>> {
    Ok(match item {
        Item::Enum(item_enum) => self.parse_enum(item_enum),
        Item::Struct(item_struct) => self.parse_struct(item_struct),
    })
}
```
Currently the function contains a bit too many levels of abstraction.

Reply:
Done, only left the use one as it's simple.